Cargo.toml

```
@@ -13,6 +13,7 @@ members = [
     "examples/ping-example",
     "examples/relay-server",
     "examples/rendezvous",
+    "examples/upnp-example",
```

I know they aren't all consistent but actually I'd like them to not repeat the name `example`.

```suggestion
    "examples/upnp",
```

---

The crate should probably still say `-example` to not cause any clashes with other crates.

---

no worries, updated :)